doc/source/known_issues.rst

```diff
@@ -1,13 +0,0 @@
-.. _known_issues:
```

Comment:
Nope, content was part of the release notes for the rc releases. It was moved the separate file once we did the GA. 

I'll recreate the file once we have more issues :) otherwise having a empty doc doesn't make much sense.